docs/topics/vscode-extension-reviewing-projects.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-reviewing-projects_{context}']
+
+= Reviewing and resolving issues with the {ProductName} extension
+
+You can review and resolve issues identified by the {ProductName} extension in the Issue Run configuration pane.
+
+== Reviewing issues
+
+The {ProductName} extension uses icons to indicate the severity of an issue and wherher or not an issue can be solved by applying Quick Fixes (automatic code replacement).
+
+.Procedure
+
+. Select a run configuration directory on the Run configuration pane and expand its folders to view the *Hints* generated for each application file.
+. Select a hint to view the source code.
+. Right-click a hint and select *View Details* to view the *Rule ID* and other information.
++
+To the left of each hint is an icon showing the severity of the issue and an indication if it is possible to apply Quick Fixes to the issue.
++
+The following icons indicate the severity and state of an issue:
++
+.{ProductName} extension issue icons
+[cols="15%a,85%",options="header",]
+|====
+|Icon |Description
+|image::vs_mandatory.png[Mandatory] |The issue must be fixed for a successful migration.
+|image::vs_optional.png[Optional or Quick Fix] |The issue is either optional to fix for migration _or_ it can be resolved by applying Quick Fixes.
+
+To determine which meaning is correct, right-click on the issue and look at the options in the list that opens. If *Apply All QuickFixes* appears, then Quick Fixes are available. Otherwise, fixing it is optional.
```

Comment:
That is weird. Why would the same icon have two different meanings?

Reply:
I'm surprised, too, but John Steele was clear that this is the case.